CDCgov/prime-reportstream
# 7050 Submission History Jurisdictional Filter data for UP

This PR makes various updates to ensure that the data that you fetch from the History API displays correctly through the Jurisdictional filtering step for the Universal Pipeline.

Test Steps:
1. Submit a Full ELR report via Postman (or whichever you use)
2. After receiving a `submissionId`, use that to check the History API until the response includes a list of `destinations`.
3. Verify that the following fields are populated for the destinations: `organization, organization_id, service, itemCount, itemCountBeforeQualityFiltering`

## Changes
- Populate `itemCountBeforeQualFilter` on various places in the database where they should be accessible.
- Update `enrichWithProcessAction` to account for the UP using `translate` instead of `process`.
- Update a related test case to improve coverage.

## Linked Issues
- #7050

## To Be Done
- #7051
